Update site.sh to only deploy root project

ibaca opened the pull request:
Hi! I have created the pull request to ask... are you using this script? does it works? or it upload the first project but maven continues and throws error in nested project? I tried it the other day and I get various errors. I thinks this should fix the problem.

TDesjardins commented:
Hello @ibaca ,

I am not using this script because I deploy on a Windows machine :-)
I simply execute the Maven command on the demo module after locally changing the outputDirectory from:
`<outputDirectory>${basedir}/gwt-ol3-demo/target/gwt-ol3-demo/demo</outputDirectory>`
to
`<outputDirectory>${basedir}/../gwt-ol3-demo/target/gwt-ol3-demo/demo</outputDirectory>`
Running site deploy on the parent module throws an error.

If your changes work for you, you could merge this pull request. If you like, you could check your script by updating the site. There are some small changes since the last update (Overviewmap in Graticule-Example).

ibaca commented:
I have modified the script, first `clean package` to compile the lib and the demo, and later deploy the site, but now only one time, I have added the `non recursive` parameter. This is a bit ugly because this deploy the parent project site with the output of this child project, but... it is quite simple and works.
